insights/parsers/tests/test_hostname.py

```diff
@@ -10,6 +10,7 @@ def test_hostname():
     assert data.fqdn == "rhel7.example.com"
     assert data.hostname == "rhel7"
     assert data.domain == "example.com"
+    assert "{}".format(data) == "<hostname: rhel7, domain: example.com>"
```

Comment:
@PaulWay in python 2.6 you need to supply the position for the arg: `"{0}".format(data)`.